### PR TITLE
feat(validator): Movie Model Validator

### PR DESCRIPTION
**Pull request by robintsunny**
what: Validator for movie model
why: to validate movie model

### CODE REVIEW

**Review by lvamaral — APPROVED** [review]

File: lib/plugins/features/movies/index.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 
   server.route([{
     method: 'POST',
     path: '/movies',
     config: {
-      handler: (request, reply) => {
-        reply('hello world');
+      handler: (request, reply) => reply('hello world'),
+      validate: {
+        payload: MovieValidator
```
Comment (vinay-pad) [inline lib/plugins/features/movies/index.js:13]: Why is the validator.js file empty? looks like nothing is being validated in this PR?

**Review by vinay-pad — COMMENTED** [review]
Just some quick feedback. Maybe your intention is to keep each PR as small as possible. I think thats a really good intention and its something we should always try to do. But i feel like the granularity of these PRs might be a bit too small. It might be fine to add some more changes in a given PR (maybe upto 50 to 70 lines should be fine). For example in this PR you can actually add the changes to validate the request payload and add corresponding tests. I think thats an acceptable level of granularity for a PR!

File: lib/plugins/features/movies/index.js (hunk 1 above)
Comment (robintsunny) [reply lib/plugins/features/movies/index.js:13]: there wasn't a validate.js file, but there is a sfmovies/lib/validators/movies.js file which is validating the Movie object. Is that what you were looking for? The tutorial did not have me create a validator.js file, or I have not reached that step yet.

Comment (vinay-pad) [reply lib/plugins/features/movies/index.js:13]: oh sorry i meant the `validators/movies.js` file which is empty.

Comment (robintsunny) [reply lib/plugins/features/movies/index.js:13]: I'm not sure why that file wasn't added during the commit, but i've added it and pushed it up

Comment (robintsunny) [reply lib/plugins/features/movies/index.js:13]: i'll also add the validation tests before merging. working on it right now

File: lib/validators/movie.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).required(),
+  release_year: Joi.number().integer().min(1878).max(9999).optional()
+});
```
Comment (mgartner) [inline lib/validators/movie.js]: new line needed

File: lib/validators/movie.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
```
Comment (mgartner) [inline lib/validators/movie.js:5]: did the guide tell you to add test for this validator?

File: lib/validators/movie.js (hunk 2 above)
Comment (robintsunny) [reply lib/validators/movie.js]: done!

File: lib/validators/movie.js (hunk 3 above)
Comment (robintsunny) [reply lib/validators/movie.js:5]: It's the next step, but i'm gonna add it to this PR to group validation together

Comment (robintsunny) [reply lib/validators/movie.js:5]: added the validation tests

**Review by mgartner — APPROVED** [review]
LGTM

**Pull request merged**